### ISSUE TITLE
is it dead?

### ISSUE THREAD

**Issue by xandro0777** [Pato05/uploadgram-app#17]
Just installed, "upload" shows no reaction, can't share files to the app, can't find any way to make anything work?

**Comment by licaon-kter** [comment]
says `dead` here https://uploadgram.me/terms.html

**Comment by Pato05** [comment]
Yes, Uploadgram is in fact dead, as also stated in the telegram announcements channel. Though if anyone is interested in self hosting, eventually the source code is going to be released and the application will also receive an update that will allow you to specify your own endpoints.

**Comment by imsweetdogs** [comment]
hi @Pato05, i don't see server soource code.

> Yes, Uploadgram is in fact dead, as also stated in the telegram announcements channel. Though if anyone is interested in self hosting, eventually the source code is going to be released and the application will also receive an update that will allow you to specify your own endpoints.
